components/overview/OverviewNav.tsx

```diff
@@ -7,6 +8,8 @@ interface Props {
 }
 
 export const OverviewNav = ({selected, setSelected}:Props) => {
```

Comment:
Happy to hear your opinions on this, but I'm somewhat against the use of 'overview' just by itself, as I feel like it doesn't communicate enough meaningful information to someone new looking at the code. Maybe something like topicOverview, topicNav, etc?

Reply:
fyi this would apply throughout - e.g, components/<newName>, en/<newName>.json, etc

Reply:
Yeah I have no issues with that! And I'm sorry, it's because at work we have a product page called "Overview" so its so ingrained to me what it means hahaha but yes, I'll fix this as much as I can